src/components/square/square.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import './square.css';
+
+const Square = props => {
+  return <div className='box'>{props.value}</div>;
```

Comment:
Do you need a key on this? Not sure. Look at the console when running it in the browser and see if there are warnings or errors.

Reply:
just assigned a key of props.value to each square